client/webserver/site/src/html/wallets.tmpl

```diff
@@ -714,7 +717,25 @@
             <td data-tmpl="voting"></td>
           </tr>
         </tbody>
-      </table>{{/* TODO: ADD MANUAL INPUT */}}
+      </table>
+      <p class="mt-3">
+        <label for="customVspUrl">Or add custom VSP URL:</label>
```

Comment:
Needs translation. Add an entry for this in `en-us.go`.

Reply:
Most of the strings in `wallets.tmpl` are not localized yet.  Maybe it's best to have a separate issue to cover all of those?  (Probably related to/overlaps with #1454, #1880?)

Reply:
It looks like just the staking related strings are not localized. I think if you wanted to do them here I think it would be fine, or it could be done in another issue as well.

Reply:
Addressed in #2544

Reply:
Ideally, every displayed string would have a translation entry, even if just en-US. But I'm as much if not more guilty than others of not doing it, so I understand and I tend to be permissive. @peterzen, I won't hold this up, but in general try to do translation and definitely do them if someone calls them out in review.

Reply:
Agreed - in this one I did the l18n separately so as not to clutter the PR with changes that are unrelated to the feature.